lib/node_provider.dart

```diff
@@ -143,9 +143,17 @@ class DgApiNodeProvider extends SimpleNodeProvider implements SerializableNodePr
 
     IOldApiConnection connection = new OldApiBaseAuthConnection(url, username, password);
 
+    var nd = new DGNode('/$n', this);
```

Comment:
Appears to be remnants from something they tried in the past. Cleaned up some of it.